### DISCUSSION TITLE
Isn't 25$/person/month a bit too much?

### DISCUSSION THREAD

**Discussion by ibeladi-mrsool** [stackaid/stackaid#3] [Q&A] [answered]
I would suggest starting with a lower minimum rate as a start. 25$ is a bit hefty for some people. I would suggest suggesting 25 but you can allow maybe as low as 5. Since it is a recurring donation, it surely would build up.

**Comment by voxpelli** [comment, marked as answer]
Its free to chose your own amount, no? Or is $15 the lowest? (I think they lowered it since you wrote this)

**Reply by ibeladi-mrsool** [reply]
Yeah, they changed it now, it used to be fixed 25$.
![image](https://user-images.githubusercontent.com/11488356/199048242-a27689f0-255b-4490-a1ee-822d6212badd.png)
